core/integration/steps/amm_submission.go

```diff
@@ -218,6 +222,13 @@ func (a ammRow) marketID() string {
 	return a.r.MustStr("market id")
 }
 
+func (a ammRow) proposedFee() num.Decimal {
+	if !a.r.HasColumn("proposed fee") {
+		return num.DecimalFromFloat(0.01)
```

Comment:
Not a fan of default values for what is a required parameter. This makes existing tests exhibit somewhat magic behaviour, and doesn't quite make sense when it comes to amending an AMM. I also think the fee parameter needs to be added to the event in the event proto, datanode, and integration tests